src/middlewared/middlewared/etc_files/local/netdata/netdata.conf

```diff
@@ -0,0 +1,55 @@
+<%
+	# We do an empty update to ensure that additional_params value is filled when we get netdata global settings
+	netdata_global_settings = middleware.call_sync('netdata.update', {})
+
+%>
+# $FreeBSD$
+
+# Netdata Configuration
+#
+# The value shown in the commented settings, is the default value.
+#
+
+# global netdata configuration
+
+[global]
+	history = ${netdata_global_settings['history']}
+	bind to = ${' '.join(netdata_global_settings['bind'])}
```

Comment:
Did you make sure it listens to localhost if someone specifies a different IP? I am thinking you will need to explicitly add it.

Reply:
Nope it doesn't. But then I figured that is what the user wants and if he is making that change, he needs to be aware of the consequences. Initially I reflected this in nginx.conf as well but then I removed that bit. However `0.0.0.0` works and there are no issues on the local host. If you think we should add localhost at all times, I am going to do that. Now that you have mentioned it, I think it does make more sense to have it here ensuring that it works with nginx

Reply:
Yes, we expose /netdata and UI uses it. If that doesn't work users will start filling bugs ; )

Reply:
Makes total sense : ) Will have it updated right away : )

Reply:
That reminds me, we should handle IPv6, so not just 0.0.0.0, but also `::`